diff --git a/dist/chart.js b/dist/chart.js
index d5f9143032183fc9a4e560c19681e883484a5a84..ca84a5a8ddb5dcba1691dd5fca6209c2b79a2ba7 100644
--- a/dist/chart.js
+++ b/dist/chart.js
@@ -3718,7 +3718,7 @@ function titleArgs(scale, offset, position, align) {
             titleX = offsetFromEdge(scale, position, offset);
         }
         titleY = _alignStartEnd(align, bottom, top);
-        rotation = position === 'left' ? -HALF_PI : HALF_PI;
+        rotation = position === 'left' ? -HALF_PI : -HALF_PI;
     }
     return {
         titleX,
